src/morphology/english/getNounForms.js

```diff
@@ -0,0 +1,88 @@
+// "use strict";
+const irregularNouns = require( "./irregularNouns.js" );
+const singularizeRegex = require( "./regexNoun.js" ).singularizeRegex;
+const pluralizeRegex = require( "./regexNoun.js" ).pluralizeRegex;
+const hispanicRegex = require( "./regexNoun.js" ).hispanicRegex;
+
+const isUndefined = require( "lodash/isUndefined.js" );
+const unique = require( "lodash/uniqBy" );
+
+const checkIrregulars = function( word ) {
```

Comment:
Please add documentation to this function and the other functions below.

Reply:
Done.